cookbook/deployment/cluster/auth_setup.py

```diff
@@ -1,8 +1,498 @@
+
+
+####################################
+Migrating Your Authentication Config
+####################################
+
+Using Okta as an example, you would have previously seen something like the following:
+
+On the Okta side:
+
+* An Application (OpenID Connect Web) for Flyte Admin itself (e.g. **0oal5rch46pVhCGF45d6**).
+* An Application (OpenID Native app) for Flyte-cli/flytectl (e.g. **0oal62nxuD6OSFSRq5d6**).
+  These two applications would be assigned to the relevant users.
+* An Application (Web) for Flyte Propeller (e.g. **0abc5rch46pVhCGF9876**).
+  This application would either use the default Authorization server, or you would create a new one.
+
+On the Admin side, you would have had the following configuration:
+
+.. code-block:: yaml
+
+    server:
+    # ... other settings
+    security:
+        secure: false
+        useAuth: true
+        allowCors: true
+        allowedOrigins:
+        - "*"
+        allowedHeaders:
+        - "Content-Type"
+        oauth:
+        baseUrl: https://dev-62129345.okta.com/oauth2/default/
+        scopes:
+            - profile
+            - openid
+            - email
+        claims:
+            iss: https://dev-62129345.okta.com/oauth2/default
+            aud: 0oal5rch46pVhCGF45d6
+        clientId: 0oal5rch46pVhCGF45d6
+        clientSecretFile: "/Users/ytong/etc/secrets/oauth/secret"
+        authorizeUrl: "https://dev-62129345.okta.com/oauth2/default/v1/authorize"
+        tokenUrl: "https://dev-62129345.okta.com/oauth2/default/v1/token"
+        callbackUrl: "http://localhost:8088/callback"
+        cookieHashKeyFile: "/Users/ytong/etc/secrets/hashkey/hashkey"
+        cookieBlockKeyFile: "/Users/ytong/etc/secrets/blockkey/blockkey"
+        redirectUrl: "/api/v1/projects"
+        thirdPartyConfig:
+            flyteClient:
+            clientId: 0oal62nxuD6OSFSRq5d6
+            redirectUri: http://localhost:12345/callback
+
+From the Flyte-cli side, these two settings were needed:
+
+.. code-block:: bash
+
+    FLYTE_PLATFORM_HTTP_URL=http://localhost:8088 FLYTE_CREDENTIALS_CLIENT_ID=0oal62nxuD6OSFSRq5d6 flyte-cli ...
+
+**FLYTE_PLATFORM_HTTP_URL** is used because **flyte-cli** uses only gRPC to communicate with Admin. It needs to know the HTTP port (which Admin hosts on a different port because of limitations of the 
+grpc-gateway library). **flyte-cli** uses this setting to talk to **/.well-known/oauth-authorization-server** to retrieve information regarding the auth endpoints.  Previously this redirected to the
+Okta Authorization Server's metadata endpoint. With this change, Admin now hosts its own (even if still using the external Authorization Service).
+
+After version `0.13.0 <https://github.com/flyteorg/flyte/tree/v0.13.0>`__ of the platform, you can still use the IdP as the Authorization Server if you so choose. That configuration would now become:
```

Comment:
Is version 0.13.0 up-to-date?

Reply:
this should be fine, since this is just noting a feature that should be supported in 0.13+